kolibri/core/assets/src/state/actions.js

```diff
@@ -223,6 +223,26 @@ function getCurrentSession(store) {
   .catch(error => { handleApiError(store, error); });
 }
 
+function getFacilityConfig(store) {
+  const coreApp = require('kolibri');
+  const FacilityCollection = coreApp.resources.FacilityResource
+    .getCollection()
+    .fetch();
+
+  return FacilityCollection.then(facilities => {
+    const currentFacilityId = facilities[0].id;
+    const facilityConfigCollection = coreApp.resources.FacilityDatasetResource
+      .getCollection({ facility_id: currentFacilityId })
+      .fetch();
+    return facilityConfigCollection
+      .then(facilityConfig => {
+        store.dispatch('CORE_SET_FACILITY_CONFIG', facilityConfig);
```

Comment:
I think it'd be best to write a mapper for the state to avoid [underscores and verbose-ness in the frontend](https://github.com/learningequality/kolibri/pull/1372/files#diff-3705966c69fdd78812ff7c734e1b92c7R32)